cdc/kv/shared_client.go

```diff
@@ -693,6 +694,11 @@ func (s *SharedClient) doHandleError(ctx context.Context, errInfo regionErrorInf
 			s.scheduleRangeRequest(ctx, errInfo.span, errInfo.subscribedTable)
 			return nil
 		}
+		if innerErr.GetCongested() != nil {
+			metricKvCongestedCounter.Inc()
+			s.scheduleRangeRequest(ctx, errInfo.span, errInfo.subscribedTable)
```

Comment:
It should be `scheduleRegionRequest` I guess.

Reply:
previous, `region_not_found` is handled by `scheduleRangeRequest`

Reply:
They are different. If the region is not found, we need to ask PD whether the region is changed or not. But for Congested, it's not necessary.